lib/charms/mongodb/v0/mongodb_provider.py

```diff
@@ -248,7 +248,7 @@ def update_app_relation_data(self) -> None:
 
```

Comment:
bump lib patch. Perhaps we should merge #266 first since we want `6/edge` to have its own LIBAPI

Reply:
Done, thank you 😄